src/webui/www/public/index.html

```diff
@@ -29,17 +30,17 @@ <h1>qBittorrent WebUI</h1>
             <img src="images/qbittorrent-tray.svg" alt="qBittorrent logo" />
         </div>
         <div id="formplace" class="col">
-            <form id="loginform" method="post" onsubmit="submitLoginForm(event);">
```

Comment:
Why delete this and use addEventListener instead?

Reply:
IMO it is cleaner to move code things to js instead of mixing html with js.